mmsoap/client.py

```diff
@@ -72,58 +55,77 @@ def create(self, object_name):
 
-        Keyword arguments:
-        send_mode -- used for testing. One of:
-            * normal (default) -- send as normal
-            * dropAll -- drop (not send) the requested messages, return
-              random errors and successes
-            * dropAllWithErrors -- drop the requested messages,
-              return errors
-            * dropAllWithSuccess -- drop the requested messages,
-              return success
+    def send_messages(self, recipients, content, send_mode='normal', scheduled=None, report=None, id=None):
         """
-        recipients_type = self.create("RecipientsType")
+        Send a single message to `recipients`.
+
+        :param recipients: Iterable of recipient numbers
+        :param content: Message content
+        :param send_mode: Used for testing. Should be one of:
+                            * normal (default) -- send as normal.
+                            * dropAll -- drop (not send) the requested messages, return a mix of errors and successes.
+                            * dropAllWithErrors -- drop the requested messages, return all as errors.
+                            * dropAllWithSuccess -- drop the requested messages, return all as success.
+        :param scheduled: Scheduled date/time of the message in UTC format.
+        :param report: Request a delivery report for this message.
+
+        :return: Response containing count of sent, scheduled, failed and any associated error code types.
+        """
+        recipients_type = self.create('RecipientsType')
 
         for recipient in recipients:
-            this_recipient = self.create("RecipientType")
+            this_recipient = self.create('RecipientType')
             this_recipient.value = recipient
             recipients_type.recipient.append(this_recipient)
 
-        message = self.create("MessageType")
+        message = self.create('MessageType')
         message.recipients = recipients_type
         message.content = content
 
-        message_list = self.create("MessageListType")
-        message_list.message = [message]
+        if id:
+            message._sequenceNumber = id
+
+        if scheduled:
+            message.scheduled = scheduled
+
+        if report:
+            message.deliveryReport = report
+
+        message_list = self.create('MessageListType')
+        message_list.message = message
         message_list._sendMode = send_mode
 
-        request_body = self.create("SendMessagesBodyType")
+        request_body = self.create('SendMessagesBodyType')
         request_body.messages = message_list
 
-        response = self.client.service.sendMessages(self.authentication,
-                                                    request_body)
+        response = self.client.service.sendMessages(self.authentication, request_body)
         self.raise_for_response(response)
 
         return response
 
-    def raise_for_response(self, response):
+    @staticmethod
```

Comment:
What is the point of this @staticmethod considering this class self references the method. Unless I"m missing something?